pkg/apis/core/validation/shoot.go

```diff
@@ -352,6 +385,14 @@ func validateKubernetesVersionUpdate(new, old string, fldPath *field.Path) field
 	return allErrs
 }
 
+func validateCri(newCri *core.Cri, oldCri *core.Cri, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if (newCri == nil || !newCri.EnableContainerD) && (oldCri != nil && oldCri.EnableContainerD) {
```

Comment:
It looks like containerd cannot be disabled once enabled for a worker.
But the reverse is possible?

Please add some docs in the `types_shoot.go` files about this, at least.

Reply:
I changed it, it can be updated.